python/tests/test_server.py

```
@@ -114,6 +114,25 @@ def test_register_same_file_reuses_token(served):
     assert len(_server._local_files) == 1
 
 
+def test_unregister_local_file_drops_the_token(served, tmp_path):
```

Ruff (0.16.2)

[warning] Unused function argument: `tmp_path`

(ARG001)